test/auth_test.rb

```diff
@@ -515,6 +515,26 @@ def app
       assert_match %r{/account/password/reset\z}, last_response.headers["Location"]
     end
 
+    it "redirects to login on forbidded responses that are not two-factor related" do
+      stub_heroku_api do
+        post("/oauth/authorizations") {
+          err = MultiJson.encode({
+            id: "forbidden",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.